### DISCUSSION TITLE
interfaces

### DISCUSSION THREAD

**Discussion by f4n0** [StefanMaron/BusinessCentral.LinterCop#276] [New Rule]
I think this would be a nice thing to have: 

1. Interfaces must start with a uppercase i (c# style)
2. Interfaces must end with v[1-9]: 
    This will help the creation of a newer versions of interfaces (deprecate the older version and create a newer one) 
    example: "IService v1". few month later, i need to add a new method so i will obsolete the old one and i create "IService v2"

**Comment by rvanbekkum** [comment]
For 1., I would say, ignore prefixes. So if your affix is "ABC" and you write "ABC_IVehicle" or "ABC IVehicle" then that would be ok.
What do you think?

**Reply by f4n0** [reply]
yes, i've missed the prefix
so ABC IService!
